auto_nag/round_robin.py

```diff
@@ -67,16 +67,25 @@ def feed(self, teams, rr=None):
         # - components -> dictionary: Product::Component -> strategy name
         # - strategies: dictionary: {calendar: url}
 
+        torm = []
+
         # Get all the strategies for each team
         for team, data in rr.items():
-            calendars = self.get_calendar(team, data)
-            self.all_calendars += list(calendars.values())
-
-            # finally self.data is a dictionary:
-            # - Product::Component -> dictionary {fallback: who to nag when we've nobody
-            #                                     calendar}
-            for pc, strategy in data["components"].items():
-                self.data[pc] = calendars[strategy]
+            try:
+                calendars = self.get_calendar(team, data)
+                self.all_calendars += list(calendars.values())
+
+                # finally self.data is a dictionary:
+                # - Product::Component -> dictionary {fallback: who to nag when we've nobody
+                #                                     calendar}
+                for pc, strategy in data["components"].items():
+                    self.data[pc] = calendars[strategy]
+            except (BadFallback, InvalidCalendar) as err:
+                logger.error(err)
```

Comment:
Are these errors reported to us via email?

Reply:
Normally yes